src/index.html

```diff
@@ -11,6 +11,34 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form 
+		  action="#" 
+		  method="post" 
+		  class="big-search"
```

Comment:
```suggestion
		  class="search"
```
please check other elements and pay attention on this point from [checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md)
![Screenshot from 2023-03-28 14-53-50](https://user-images.githubusercontent.com/33029864/228228176-6b7d2b62-b3f9-4fb8-b2dd-e35a861fc3ab.png)